portfolio/src/main/java/com/google/sps/servlets/LikeServlet.java

```diff
@@ -39,17 +39,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     int retries = 3;
     while (true) {
       Transaction txn = datastore.beginTransaction();
-      Entity commentEntity;
       try {
-        commentEntity = datastore.get(likeKey);        
+        buildTransaction(datastore, likeKey, txn);
       } catch (EntityNotFoundException e) {
         response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Comment not found.");
         break;
       }
-      long count = (long) commentEntity.getProperty("likeCount");
-      ++count;
-      commentEntity.setProperty("likeCount", count);
-      datastore.put(txn, commentEntity);
```

Comment:
Nit: This shouldn't be part of this PR.

Reply:
emmm looks like I pushed to the wrong PR 😅